agent/pom.xml

```diff
@@ -16,14 +16,10 @@
     limitations under the License.
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.1.0 http://maven.apache.org/maven-v4_1_0.xsd">
```

Comment:
I think the correct location for the XSD is https://maven.apache.org/xsd/maven-4.1.0-beta-4.xsd. The current URL gives a HTML page with "Not found".

Reply:
That's right, it needs to be fixed.
However, I think we did set up a redirection and `https://maven.apache.org/xsd/maven-4.1.0.xsd` will redirect to latest beta.  I can't find out where this redirect source is...

Reply:
So `http://maven.apache.org/maven-v4_0_0.xsd` also redirects to `https://maven.apache.org/xsd/maven-4.0.0.xsd` ...

Reply:
https://github.com/apache/maven-site/tree/master/content/resources/xsd